static/js/containers/CopyLinkDialog.js

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import { createSimpleActionHelpers } from "../lib/redux"
+import { setDialogVisibility } from "../actions/signup_dialog"
+import { connect } from "react-redux"
+import Dialog from "material-ui/Dialog"
+import { dialogActions } from "../components/inputs/util"
+
+const dialogStyle = {
+  maxWidth: "500px"
+}
+class CopyLinkDialog extends React.Component {
+  props: {
+    open: boolean,
+    setDialogVisibility: (b: boolean) => void
+  }
+
+  copyToClipboard = e => {
+    this.textArea.select()
+    document.execCommand("copy")
+    e.target.focus()
+  }
+
+  render() {
+    const { open, setDialogVisibility } = this.props
+    return (
+      <Dialog
+        title="Share Link to Record"
+        titleClassName="dialog-title"
+        contentClassName="dialog share-dialog"
+        className="signup-dialog-wrapper"
+        open={open}
+        onRequestClose={() => setDialogVisibility(false)}
+        contentStyle={dialogStyle}
+        autoScrollBodyContent={true}
+      >
+        <p>
+          Copy this link to share with a university, employer or anyone else of
+          you choosing. Anyone you share this link with will have access to your
+          record forever.
+        </p>
+        <div className="share-form-wrapper">
+          <form className="share-url">
+            <textarea
```

Comment:
Small thing: can we make this an input instead of a textarea? It looks cleaner, and I think edX is using an input. (See [screenshot](https://images.zenhubusercontent.com/56c1e5a26caf56c9d38c8552/42cb8d6b-04ba-443a-a3b1-3cb7ca26c725))